src/renderer/components/+add-cluster/add-cluster.tsx

```diff
@@ -119,6 +118,11 @@ export class AddCluster extends React.Component {
     }
   };
 
+  onDropKubeConfig = (files: File[]) => {
```

Comment:
We could show error if the user tried to drag & drop multiple files. I suppose this could be done in DropFileInput too.

Reply:
D&D for multiple files is supported, why not?

Reply:
In this function it seems only one dropped file is actually used?

Reply:
Yes, right, for adding clusters expected only single kube-config file in d&d, so first one is used by default I would say. 
There is no error to handle imho since in UI expected as well single config file path.